tools/codegen/model.py

```diff
@@ -493,9 +493,14 @@ def __post_init__(self) -> None:
     def signature(self) -> 'FunctionSchema':
         return self.out.func.signature()
 
-    def functions(self) -> Iterator[NativeFunction]:
-        yield self.out
-        yield self.functional
+    def functions(self, *, functional_first: bool = False) -> Iterator[NativeFunction]:
```

Comment:
Kind of goofy

Reply:
yeah you're right, I think we should be able to unconditionally use the functional kernel first.